a10_octavia/common/utils.py

```diff
@@ -88,22 +89,28 @@ def convert_to_hardware_thunder_conf(hardware_list):
     hardware_dict = {}
     for hardware_device in hardware_list:
         hardware_device = validate_params(hardware_device)
-        if hardware_dict.get(hardware_device['project_id']):
-            raise cfg.ConfigFileValueError('Supplied duplicate project_id ' +
-                                           hardware_device['project_id'] +
-                                           ' in [hardware_thunder] section')
+        project_id = hardware_device['project_id']
+        if hardware_dict.get(project_id):
+            raise cfg.ConfigFileValueError('Supplied duplicate project_id {} '
+                                           ' in [hardware_thunder] section'.format(project_id))
         hardware_device['undercloud'] = True
         if hardware_device.get('interface_vlan_map'):
             hardware_device['device_network_map'] = validate_interface_vlan_map(hardware_device)
             del hardware_device['interface_vlan_map']
+        hierarchical_mt = hardware_device.get('hierarchical_multitency')
```

Comment:
spelling of `hierarchical_multitency` --> `hierarchical_multitenancy`

Reply:
Yikes thanks for catching this. I had typed it in wrong in my config and was copying/pasting out from that